src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -10,19 +10,20 @@ include (GenerateExportHeader)
 if($ENV{__BuildArch} STREQUAL x86 OR $ENV{__BuildArch} STREQUAL x64)
     set(NATIVECOMPRESSION_SOURCES
         zlib-intel/adler32.c
+        zlib-intel/compress.c
         zlib-intel/crc_folding.c
         zlib-intel/crc32.c
+        zlib-intel/deflate_medium.c
         zlib-intel/deflate_quick.c
         zlib-intel/deflate.c
-        zlib-intel/fill_window_sse.c
+        zlib-intel/inffast.c
+        zlib-intel/inflate.c
+        zlib-intel/inftrees.c
         zlib-intel/match.c
+        zlib-intel/slide_sse.c
         zlib-intel/trees.c
         zlib-intel/x86.c
         zlib-intel/zutil.c
-        zlib/compress.c 
-        zlib/inffast.c
-        zlib/inflate.c
-        zlib/inftrees.c
```

Comment:
Why were these previously zlib when the rest were zlib-intel?  Is it that these files weren't previously specialized in the Intel implementation, and now they are?

Reply:
Yes, we added new optimizations for decompression in the latest ZLib-Intel. This was not available previously.